### PR TITLE
version

### PR DESCRIPTION
**Pull request by mattobviously**

### CODE REVIEW

File: app/views/table_on_steroids/_table_on_steroids.html.erb [hunk 1]
```diff
@@ -70,16 +71,14 @@
...
+            <%= v[:label] %>
             <% if v[:type] == 'order' %>
-              <label>
-                <%= v[:label] %>
-                <%= radio_button('knowledge', 'order', k, onchange: "$('#knowledge_base_filters#{table_on_steroid_id}').submit();", checked: params[:knowledge] && params[:knowledge][:order] == k, form: "knowledge_base_filters#{table_on_steroid_id}" ) %>
-                v
-              </label>
-            <% else %>
-              <%= v[:label] %>
+              <%= image_tag "up_arrow_white.svg", :style => "margin-left:5px;margin-right:5px;margin-bottom:5px;height:8px;cursor:pointer", onclick: "$('#knowledge_order#{table_on_steroid_id}').val('" + k + "');$('#knowledge_ascending#{table_on_steroid_id}').val('true');$('#knowledge_base_filters#{table_on_steroid_id}').submit();" %>
```
Comment (letsbreelhere) [inline app/views/table_on_steroids/_table_on_steroids.html.erb]: Any way to make these styles non-inline?

**Review by letsbreelhere — DISMISSED** [review]

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,9 @@
 
 # rspec failure tracking
 .rspec_status
+
+/.DS_Store
```
Comment (letsbreelhere) [inline .gitignore]: `**/.DS_Store` should cover all of these

File: app/views/table_on_steroids/_table_on_steroids.html.erb (hunk 1 above)
Comment (mattobviously) [reply app/views/table_on_steroids/_table_on_steroids.html.erb]: 👍

**Comment by marieke-obviously** [comment]
@mattobviously  add doc

**Review by hanaiapa — APPROVED** [review]

**Pull request merged**